source-sqlserver/datatypes_test.go

```diff
@@ -45,7 +45,7 @@ func TestDatatypes(t *testing.T) {
 		{ColumnType: `image`, ExpectType: `{"type":["string","null"],"contentEncoding":"base64"}`, InputValue: []byte{0x12, 0x34, 0x56, 0x78}, ExpectValue: `"EjRWeA=="`},
 
 		{ColumnType: `date`, ExpectType: `{"type":["string","null"],"format":"date"}`, InputValue: `1991-08-31`, ExpectValue: `"1991-08-31"`},
-		{ColumnType: `time`, ExpectType: `{"type":["string","null"],"format":"time"}`, InputValue: `12:34:54.125`, ExpectValue: `"12:34:54.125"`},
+		{ColumnType: `time`, ExpectType: `{"type":["string","null"]}`, InputValue: `12:34:54.125`, ExpectValue: `"12:34:54.125"`},
```

Comment:
Am I missing something? I don't see the actual code change that would be reflected in tests, only updates to the test?

Reply:
Whoops, this is why code review is always a good idea. Yeah, I must have somehow forgotten to add the change to the actual discovery type mapping table.

Reply:
This should be fixed now.